pkg/operator/sync.go

```diff
@@ -605,6 +605,15 @@ func (optr *Operator) syncRequiredMachineConfigPools(_ *renderConfig) error {
 				return false, nil
 			}
 
+			updating := isPoolStatusConditionTrue(pool, mcfgv1.MachineConfigPoolUpdating)
+			if updating {
+				glog.Infof("pool %s is still updating, Status: (pool updating: %v total: %d, ready %d, updated: %d, unavailable: %d)", pool.Name, updating, pool.Status.MachineCount, pool.Status.ReadyMachineCount, pool.Status.UpdatedMachineCount, pool.Status.UnavailableMachineCount)
+				syncerr := optr.syncUpgradeableStatus()
```

Comment:
As a general question, why sync the upgradeable status here in a per-pool loop? The `syncUpgradeableStatus()` already does so per pool.

We also call this right after this loop (once master pool completes) and then again at https://github.com/openshift/machine-config-operator/blob/889af725b2dc5e7503c5b0cb9b0b710fd62da967/pkg/operator/sync.go#L91. Do you think it'd be worth it to consolidate some of those calls?

Reply:
I too think that syncUpgradeableStatus() shouldn't be needed here as we will anyway sync after we iterate over all pools.

Reply:
Agree will remove